File: euler_5/euler_5.js
var answer = 0, test = 2520;

function test_num(num) {
    for (i=2; i < 21; i = i + 1) {
        if (num % i != 0) {
            return false;
        }
    }
    return true;
}

while (!answer) {
    if (test_num(test)) {
        answer = test;
    }
    test += 20;
}

console.log("The answer is: " + answer);

File: euler_5/euler_5.rb
answer = nil
test = 2520

def test_num num
  (2...20).each do |i|
    return false if num % i != 0
  end
  true
end

while !answer
  answer = test if test_num test
  test = test + 20
end

puts "The answer is: #{answer}"
